src/app/common/components/DarkModeSwitcher/DarkModeSwitcher.tsx

```diff
@@ -17,8 +18,8 @@ const DarkModeSwitcher = ({ style }: { style?: any }) => {
 
   return (
     <DarkModeButton onClick={() => applicationStore.switchDarkMode()} aria-label="Switch Dark/Light Mode" style={style ?? {}}>
-      {!applicationStore.isDarkMode && <Brightness4Icon style={{ color: 'white' }} /> }
-      {applicationStore.isDarkMode && <Brightness7Icon style={{ color: 'white' }} /> }
+      {!applicationStore.isDarkMode && <img width={34} height={28} src="/images/toggle/light.svg" alt="Copy" /> }
+      {applicationStore.isDarkMode && <img width={34} height={28} src="/images/toggle/dark.svg" alt="Copy" /> }
```

Comment:
Both lines are identical so you can just do ternary operator and use one line

Reply:
done